docs/contributing/reporting-bugs.md

```diff
@@ -146,9 +145,9 @@ renamed or resolved.
     don't know what area labels to use.
   - You can add the “help wanted” label (and claim the issue if you like) if
     that is appropriate based on the discussion. Note that sometimes we won't
-    mark a reproducible bug as “help wanted”, e.g., if we want a core
-    contributor to take it on, if the fix will happen as part of a larger
-    project, etc.
+    mark a reproducible bug as “help wanted” if we want a core
+    contributor to take it on, or if the fix will happen as part of a larger
+    project.
```

Comment:
How about: "Note that sometimes we won't mark a reproducible bug as "help wanted" for various reasons. For example, we might want a core contributor to take it on, or we expect the fix to happen as part of a larger project."